test/integration/helpers/integration-test-editor.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	BlockEditorKeyboardShortcuts,
+	BlockEditorProvider,
+	BlockList,
+	BlockTools,
+	BlockInspector,
+	WritingFlow,
+	ObserveTyping,
+} from '@wordpress/block-editor';
+import { Popover, SlotFillProvider } from '@wordpress/components';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { ShortcutProvider } from '@wordpress/keyboard-shortcuts';
+import '@wordpress/format-library';
+import {
+	createBlock,
+	unregisterBlockType,
+	getBlockTypes,
+} from '@wordpress/blocks';
+
+// Polyfill for String.prototype.replaceAll until CI is runnig Node 15 or higher.
+if ( ! String.prototype.replaceAll ) {
+	String.prototype.replaceAll = function ( str, newStr ) {
+		// If a regex pattern
+		if (
+			Object.prototype.toString.call( str ).toLowerCase() ===
+			'[object regexp]'
+		) {
+			return this.replace( str, newStr );
+		}
+
+		// If a string
+		return this.replace( new RegExp( str, 'g' ), newStr );
+	};
+}
+
+export function registerAllCoreBlocks() {
+	registerCoreBlocks();
+}
+
+export function unRegisterAllBlocks() {
+	getBlockTypes().forEach( ( { name } ) => unregisterBlockType( name ) );
+}
+
+export function createTestBlock( name, attributes ) {
+	return createBlock( name, attributes || {} );
+}
```

Comment:
```suggestion
export function createTestBlock( name, attributes, innerBlocks ) {
	return createBlock( name, attributes || {}, innerBlocks );
}
```

I guess you may as well include the 3rd argument to [createBlock](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-blocks/#createblock)

Reply:
Done.